### ISSUE TITLE
.jobs() is Deprecated

### ISSUE THREAD

**Issue by cstaykov** [fabric8io/kubernetes-client#3309]
I am using the latest version 5.5.0 from maven and I am attempting to create a Job using:

```
            Job job = new JobBuilder()
                    .withApiVersion("batch/v1")
                    .withNewMetadata()
                    ...
                    .endSpec()
                    .build();
```

And then doing `job = client.batch().jobs().create(job);` but my IDE is warning me that jobs() is deprecated. Upon checking 

```
public interface BatchAPIGroupDSL extends Client {
    /** @deprecated */
    @Deprecated
    MixedOperation<Job, JobList, ScalableResource<Job>> jobs();
```
I see the annotation. However, all the documentation on how to work with Jobs on Github says to use `client.batch().jobs().` What am I supposed to use now that it is deprecated?

**Comment by rohanKanojia** [comment]
@cstaykov : Thanks for reporting. `client.batch().jobs()` was deprecated in favor of `client.batch().v1().jobs()`:
```
client.batch().v1().jobs().inNamespace("default").createOrReplace(job)
```
By documentation are you mentioning CHEATSHEET[0]. I think we should update it with the new job DSL method. Would it be possible for you to submit a PR to fix this?

[0] https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md#job

**Comment by cstaykov** [comment]
Yes, I was referring to the CHEATSHEET as well as: https://github.com/fabric8io/kubernetes-client#working-with-extensions.

I could do this, but it seems like someone else already picked it up.

**Comment by rohanKanojia** [comment]
@laxmikantbpandhare : Since you're also planning to work on https://github.com/fabric8io/kubernetes-client/issues/2887 . Is it okay if I assign this issue to Chris?

**Comment by laxmikantbpandhare** [comment]
> @laxmikantbpandhare : Since you're also planning to work on #2887 . Is it okay if I assign this issue to Chris?

Yes, that will be fine. No issues at all.

**Comment by rohanKanojia** [comment]
@cstaykov: You can take this up now :-) . I think it would be better since you found out outdated documentation

**Comment by cstaykov** [comment]
Thanks, can you assign me, as I don't have permission to?

**Comment by Tariqs** [comment]
If you are using fabrica8  kubernetes-client version 6.0.0 or above then please use the following code to create job

`Job myjob = client.batch().v1().jobs().inNamespace("default").resource(job).create();`

**Comment by manusa** [comment]
> If you are using fabrica8 kubernetes-client version 6.0.0 or above then please use the following code to create job
> 
> `Job myjob = client.batch().v1().jobs().inNamespace("default").resource(job).create();`

the deprecation on this issue is not related with the deprecation of the `create(Resource)` method.
